.ng-dev/release.ts

```diff
@@ -9,8 +9,15 @@ const npmPackages = Object.keys(releasePackages);
 /** Configuration for the `ng-dev release` command. */
 export const release: ReleaseConfig = {
   npmPackages,
-  buildPackages,
-  releaseNotes: {},
+  buildPackages: () => buildPackages(),
+  releaseNotes: {
+    groupOrder: [
```

Comment:
Shouldn’t we add all the package names here?

Reply:
Since these are the only ones ordered, they are the only ones that need to be included.  The rest are just sorted alphabetically.